### ISSUE TITLE
SET GLOBAL wsrep_provider_options='pc.weight=<n>'  not set properly, always one behind. Needs to be set twice.

### ISSUE THREAD

**Issue by claudionanni** [codership/galera#446]
On multiple MariaDB Server versions (5.5.x & 10.0.x).
If I try to set SET GLOBAL  wsrep_provider_options='pc.weight=<n>' on a cluster with more than one running node the value is not picked up immediately, it looks like it is always one behind.

SHOW GLOBAL VARIABLES like 'wsrep_provider%';
....pc.weight = 1; ....

SET GLOBAL  wsrep_provider_options='pc.weight=5';
SHOW GLOBAL VARIABLES like 'wsrep_provider%';
....pc.weight = 1; ....

SET GLOBAL  wsrep_provider_options='pc.weight=9';
SHOW GLOBAL VARIABLES like 'wsrep_provider%';
....pc.weight = 5; ....

SET GLOBAL  wsrep_provider_options='pc.weight=22';
SHOW GLOBAL VARIABLES like 'wsrep_provider%';
....pc.weight = 9; ....

So basically to set the parameter I have to set it twice to the same value:

SET GLOBAL  wsrep_provider_options='pc.weight=45';
SET GLOBAL  wsrep_provider_options='pc.weight=45';
SHOW GLOBAL VARIABLES like 'wsrep_provider%';
....pc.weight = 45; ....

Please note: This does not happen if only one Galera node is running.

**Comment by mkaruza** [comment]
Duplicate. Main issue is #505
